packages/app/src/cli/services/app/config/link.ts

```diff
@@ -207,3 +216,24 @@ const getAccessScopes = (appConfiguration: AppConfiguration, remoteApp: Organiza
     }
   }
 }
+
+export async function getRemoteAppConfig(apiKey: string): Promise<Partial<CurrentAppConfiguration>> {
+  const token = await ensureAuthenticatedPartners()
+  const remoteSpecifications = await fetchAppExtensionRegistrations({token, apiKey})
+  const appAccessModule = remoteSpecifications.app.extensionRegistrations.find((extension) => {
+    return extension.type.toLowerCase() === APP_ACCESS_IDENTIFIER
+  })
+  const appConfig: Partial<CurrentAppConfiguration> = {}
+
+  if (appAccessModule?.activeVersion?.config) {
+    try {
+      const appAccessConfig = JSON.parse(appAccessModule.activeVersion.config)
+      appConfig.access = appAccessConfig.access
+      // eslint-disable-next-line no-catch-all/no-catch-all
+    } catch (error) {
+      // Ignore errors
```

Comment:
do we usually silently fail? 😅

Reply:
Good question. After a quick search of the repo, it looks like there are a handful of the following:
- `catch () { return }`
- `catch () { return false }`
- `catch () { return undefined }`

The `getRemoteAppConfig` does the following:
1. calls `ensureAuthenticatedPartners`
1. calls `fetchAppExtensionRegistrations`
1. finds the `app_access` extension
1. declares an `appConfig` variable set to an empty object (`{}`)
1. parses the `appAccessModule.activeVersion.config`
   - if it successfully parses `appAccessModule.activeVersion.config`, it adds the `access` property to the `appConfig` object
1. returns the `appConfig` object

How do you feel about calling `outputDebug`/`outputWarn` when we fail to parse `appAccessModule.activeVersion.config`? 

For example:
```ts
outputDebug('Unable to parse the active access config. Please check the `[access]` section of your app config file (shopify.app.toml)'.)
```